Resources/Private/Templates/Backend/Search/InfoModule/Index.html

```diff
@@ -324,7 +324,7 @@ <h3 class="panel-title">
                                                 <f:link.action action="requeueDocument" arguments="{uid: document.uid, type: document.type}"
                                                                controller="Backend\Search\IndexQueueModule"
                                                                pluginName="searchbackend_SolrIndexqueue"
-                                                               additionalParams="{M:'searchbackend_SolrIndexqueue',route: 'searchbackend_SolrIndexqueue'}" title="ReQueue for all languages">
```

Comment:
This is probably not needed anymore, as the M parameter is not needed any the URL is properly built from the backend.